app/mainwindow2.h

```diff
@@ -91,6 +91,8 @@ public slots:
 
     void importMovie();
 
+    void lockWidgets( bool shouldLock );
```

Comment:
StackOverflow https://stackoverflow.com/questions/35879514/public-functions-versus-public-slots mentions that prior to Qt5, slots were the only way to communicate with signals, now you can just use normal functions, so my guess is that it's just syntactic sugar.